src/tetris/brick/brickFactory.ts

```diff
@@ -0,0 +1,155 @@
+/// <reference path="../../../definitions/phaser.d.ts"/>
+
+
+import "phaser";
+import Vector2 = Phaser.Math.Vector2;
+import Brick from "tetris/brick/brick";
+import BiasEngine from "tetris/biasEngine/biasEngine";
+import Field from "tetris/field/field";
+
+export default class BrickFactory {
+
+	//region public members
+	//endregion
+
+	//region public methods
+	public newBrick(field: Field): Brick {
+		const blockAssetId = this._selectBlockAssetId();
+		const bias = this._biasEngine.newBrickBias(field);
+		return this._newBrick(blockAssetId, bias.position, bias.chances);
+	}
+	//endregion
+
+	//region constructor
+	public constructor(biasEngine: BiasEngine) {
+		this._biasEngine = biasEngine;
+
+		this._brickCreationFunctions = [
+			this._newI,
+			this._newO,
+			this._newL,
+			this._newJ,
+			this._newT,
+			this._newS,
+			this._newZ
+		];
+
+		this._blockAssetIds = [
+			/*
+			TODO: add block asset ids
+			 */
+		]
+	}
+	//endregion
+
+	//region private members
+	private _biasEngine: BiasEngine;
+	private readonly _brickCreationFunctions: ((blockAssetId: string, position: Vector2) => Brick)[];
```

Comment:
Use a typescript interface for that. https://www.typescriptlang.org/docs/handbook/interfaces.html